.travis.yml

```diff
@@ -4,6 +4,8 @@ php:
   - 5.5
   - 5.6
 
+sudo: true
```

Comment:
I know :)
Not saying it is an immediate issue. Just keep it on your mind. It's more important to cache the Tomcat/Solr installation on Travis / check for it's existence. 20 minutes to run our few tests is simply unacceptable.